error-prone-contrib/src/main/java/tech/picnic/errorprone/bugpatterns/MethodReferenceUsage.java

```diff
@@ -196,6 +196,8 @@ private static ImmutableList<Name> getVariables(LambdaExpressionTree tree) {
     return tree.getParameters().stream().map(VariableTree::getName).collect(toImmutableList());
   }
 
+  // XXX: Resolve this suppression.
```

Comment:
I also tried 😓 

Do we usually create tickets for these ?

Reply:
The `MethodReferenceUsage` check is not enabled currently. It requires some more big hurdles before we can enable this. So we probably want to defer this task till we start working on this check again :).